en/docs/Administer/ProductAdministration/revoke-oauth2-application.md

```diff
@@ -1,31 +1,44 @@
 # Revoke OAuth2 Application
 
-1.  An OAuth client is created when an application [access token is generated](https://docs.wso2.com/display/AM200/Working+with+Access+Tokens) . When a subscriber creates an application and generates an access token to the application using the API Store, the Store makes a call to the API Gateway, which in turn connects with the Key Manager to create an OAuth client and obtain an access token. Similarly, to validate a token, the API Gateway calls the Key Manager, which fetches and validates the token details from the database.
+1.  An OAuth client is created when an application [access token is generated](https://docs.wso2.com/display/AM200/Working+with+Access+Tokens). When a subscriber creates an application and generates an access token to the application using the API Store, the Store makes a call to the API Gateway, which in turn connects with the Key Manager to create an OAuth client and obtain an access token. Similarly, to validate a token, the API Gateway calls the Key Manager, which fetches and validates the token details from the database.
```

Comment:
Is the URL correct?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/740